awx/main/management/commands/register_peers.py

```diff
@@ -0,0 +1,30 @@
+from django.core.management.base import BaseCommand, CommandError
```

Comment:
how do we intend to use this command?

Reply:
I'm imagining this command being used every place the existing `provision_instance` command is.  So, primarily the installer.  I know we just went over how we do have auto-discovery machinery active in `awx/main/tasks.py`, but the setting that enables it can be toggled off and some means of getting the data into the database is still required.  Or, that machinery can be preempted by having the data already entered in.